sources/arch/x86_64/interrupts.c

```diff
@@ -38,12 +38,35 @@ static char *_exception_messages[32] = {
     "reserved",
 };
 
+void dump_register(struct interrupt_stackframe const *stackframe)
+{
+    uint64_t CR2 = 0;
+
+    asm volatile("mov %%cr2, %0"
+                 : "=r"(CR2));
+    uint64_t CR3 = 0;
+
+    asm volatile("mov %%cr3, %0"
+                 : "=r"(CR3));
+
+    log("rax: 0x{pa} | rbx: 0x{pa} | rcx: 0x{pa}", stackframe->rax, stackframe->rbx, stackframe->rcx);
```

Comment:
{p} should be aligned by default

Reply:
Also we should probably follow the libfmt syntax https://fmt.dev/latest/syntax.html

Reply:
> {p} should be aligned by default

Mais dans libfmt je crois pas que ce soit aligné par défaut